adc/onboard_temperature/onboard_temperature.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include "pico/stdlib.h"
+#include "hardware/adc.h"
+
+/* References for this implementation:
+ * raspberry-pi-pico-c-sdk.pdf, page 68. 
```

Comment:
That PDF is regularly updated, which means the page-numbers shift around. Much better to reference a section-number instead :slightly_smiling_face: